.grunt-config/webpack.packages.js

```diff
@@ -87,3 +89,57 @@ module.exports = {
 	dev: devConfig,
 	prod: prodConfig,
 };
+
+function getPackagesBasedOnProjectDeps() {
```

Comment:
"getPackagesBasedOnProjectDeps" is a bit confusing, maybe just describing the function output?: "getElementorPackagesMainFiles" or something like that?

Reply:
Changed